ngsi_pm_sh/conda.recipe/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -ex
+
+mkdir -p $PREFIX/bin
+
+for f in $(ls bin); do
+    (
+        echo "#!$PREFIX/bin/bash"
+        echo "export PATH=$PREFIX/bin:/usr/bin"
+        cat bin/$f
+    ) > $PREFIX/bin/$f
+done
```

Comment:
Are you missing a newline at the end of this file?

Reply:
Including `/usr/bin` suggests that your are not including `coreutils` in your runtime dependencies.

Reply:
`xargs` and `diff` did not work. `coreutils` is in the run time dependencies in the meta.yaml file if thats what you mean.

Not having this caused the commands to return a `command not found` when the path is just `$PREFIX/bin`.

Reply:
> Are you missing a newline at the end of this file?

Added

Reply:
I had no idea that `diff` and `xargs` were missing from the conda ecosystem.